custom/icds_reports/ucr/data_sources/ag_care_monthly.json

```diff
@@ -0,0 +1,715 @@
+        "comment": null
+      },
+      {
+        "display_name": null,
+        "datatype": "string",
+        "type": "expression",
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": true,
+        "expression": {
+          "location_id": {
+            "expression": {
+              "datatype": null,
+              "type": "property_name",
+              "property_name": "owner_id"
+            },
+            "type": "root_doc"
+          },
+          "location_type": "state",
+          "location_property": "_id",
+          "type": "ancestor_location"
+        },
+        "create_index": true,
+        "column_id": "state_id",
+        "comment": null
+      },
+      {
+        "display_name": null,
+        "datatype": "date",
+        "type": "expression",
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": false,
+        "expression": {
+          "type": "named",
+          "name": "display_month"
+        },
+        "column_id": "month",
+        "create_index": false,
+        "comment": null
+      },
+      {
+        "display_name": null,
+        "datatype": "string",
+        "type": "expression",
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": false,
+        "expression": {
+          "type": "named",
+          "name": "month_start"
+        },
+        "column_id": "month_start",
+        "create_index": false,
+        "comment": null
+      },
+      {
+        "display_name": null,
+        "datatype": "string",
+        "type": "expression",
+        "is_primary_key": false,
+        "transform": {},
+        "is_nullable": false,
+        "expression": {
+          "type": "named",
+          "name": "month_end"
+        },
+        "column_id": "month_end",
+        "create_index": false,
+        "comment": null
+      },
+      {
+        "filter": {
+          "type": "named",
+          "name": "open_in_month",
+          "comment": null
+        },
+        "display_name": null,
+        "type": "boolean",
+        "column_id": "open_in_month",
+        "comment": null
+      },
+      {
+        "filter": {
+          "type": "and",
+          "filters": [
+            {
+              "type": "named",
+              "name": "open_in_month",
+              "comment": null
+            },
+            {
+              "operator": "gte",
+              "expression": {
+                "type": "named",
+                "name": "hcm_days"
+              },
+              "type": "boolean_expression",
+              "property_value": 21,
+              "comment": null
+            }
+          ]
+        },
+        "display_name": null,
+        "type": "boolean",
+        "column_id": "hcm_21_plus_days",
+        "comment": null
+      },
+      {
+        "filter": {
+          "type": "and",
+          "filters": [
+            {
+              "type": "named",
+              "name": "open_in_month",
```

Comment:
most dashboard indicators filter for open, not migrated, and availing. So it is possible this will vary from similar dashboard ones

Reply:
Akshita/Sunaina fyi